pkg/sdk/testint/storage_integration_gen_integration_test.go

```diff
@@ -215,7 +215,17 @@ func TestInt_StorageIntegrations(t *testing.T) {
 	}
 
 	t.Run("Create - S3", func(t *testing.T) {
-		id := createS3StorageIntegration(t)
+		id := createS3StorageIntegration(t, sdk.RegularS3Protocol)
+
+		storageIntegration, err := client.StorageIntegrations.ShowByID(ctx, id)
+		require.NoError(t, err)
+
+		assertStorageIntegrationShowResult(t, storageIntegration, id, "some comment")
+	})
+
+	t.Run("Create - S3GOV", func(t *testing.T) {
```

Comment:
Do we have any plans to test on China deployment?

Reply:
Not sure, I guess for now there's no need, so I would stay with only leaving this one as we are capable of testing S3GOV storage integration right now

Reply:
We may track it in a separate ticket if we want.

Reply:
Added to SNOW-1820099